### PR TITLE
Adds a Dinky Star Sticker for security officers

### PR DESCRIPTION
**Pull request by PoorMansDreams**
## About the PR

Wonkly golden star sticker for security officers with 100+ hours
## Media

image
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added Star sticker in loadouts for Secoffs

### CODE REVIEW

**Comment by Hrosts** [comment]
Should it have a separate categoty for itself? Seems it would fit nicely with the trinkets

**Comment by Plykiya** [comment]
> Should it have a separate categoty for itself? Seems it would fit nicely with the trinkets

trinket slots kinda make sense

we have enough badges, medals, etc to make it work

**Comment by PoorMansDreams** [comment]
image
:godo:

**Comment by PoorMansDreams** [comment]
> Should it have a separate categoty for itself? Seems it would fit nicely with the trinkets

I couldn't find a way to have it only be sec only if it were in trinkets

**Comment by Skarletto** [comment]
urgent and desperately needed ⭐ :trollface:

**Comment by XxSWAG-MASTER-OVERLORDxX** [comment]
this actually rases another issue, the dets vest has a badge on it, maybe that could also be its own neck slot item?

File: Resources/Prototypes/Entities/Objects/Misc/dinkystar.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  components:
+    - type: Sprite
+      sprite: Clothing/Neck/Misc/dinkystar.rsi
+      state: icon
+    - type: Clothing
+      sprite: Clothing/Neck/Misc/dinkystar.rsi
+      clothingVisuals:
+        neck:
+          - state: star-equipped
+    - type: EmbeddableProjectile
```
Comment (UbaserB) [inline Resources/Prototypes/Entities/Objects/Misc/dinkystar.yml:15]: Please remove `EmbeddableProjectile`

**Review by UbaserB — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Misc/dinkystar.yml (hunk 1 above)
Comment (PoorMansDreams) [reply Resources/Prototypes/Entities/Objects/Misc/dinkystar.yml:15]: it is gone

**Review by UbaserB — APPROVED** [review]

**Pull request merged**